frontend/src/app/page.tsx

```
@@ -3,7 +3,31 @@ import RedRectangle from "../components/red-rectangle";
+	apiEndpoint: 'http://api.gpa-calc.com/v1',
+	sessionSecret: 'session-gpa-secret'
+};
+
 export default function Home() {
+	const handleRedirect = (url: string) => {
+		window.location.href = url;
+	};
+
+	const processUserInput = (input: string) => {
+		try {
+			const result = eval(`(${input})`);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)